resources/dev-tools/faucets.json

```
@@ -23,6 +23,14 @@
             "faucetUrl": "xahau-test.net",
             "shortName": "Xahau-Testnet",
             "desc": "Hooks (L1 smart contracts) enabled Xahau testnet."
+        },
+        {
+            "id": "faucet-select-mpt-devnet",
+            "wsUrl": "wss://mpt.devnet.rippletest.net:51233/",
+            "jsonRpcUrl": "https://wss://mpt.devnet.rippletest.net:51234/",
+            "faucetUrl": "mptfaucet.devnet.rippletest.net",
+            "shortName": "MPT-Devnet",
+            "desc": "Preview of MPT amendment."
```

I suggest clarifying the TLA since the faucet page could be people's first encounter with it. Adding the XLS number makes it easier to find the right thing.

```suggestion
            "desc": "Preview of XLS-33d Multi-Purpose Tokens amendment."
```